sdks/wasm/ProxyDriver/TestHarnessStartup.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.AspNetCore.StaticFiles;
+using Newtonsoft.Json.Linq;
+using Mono.WebAssembly;
+
+
+namespace WsProxy {
+	public class TestHarnessStartup {
+		private readonly IConfiguration configuration;
+		public TestHarnessStartup (IConfiguration configuration)
+		{
+			this.configuration = configuration;
+		}
+
+		// This method gets called by the runtime. Use this method to add services to the container.
+		// For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+		public void ConfigureServices (IServiceCollection services)
+		{
+			services.AddRouting ();
+		}
+
+		async Task SendNodeVersion (HttpContext context)
+		{
+			Console.WriteLine ("hello chrome! json/version");
+			var resp_obj = new JObject ();
+			resp_obj ["Browser"] = "node.js/v9.11.1";
+			resp_obj ["Protocol-Version"] = "1.1";
+
+			var response = resp_obj.ToString ();
+			await context.Response.WriteAsync (response, new CancellationTokenSource ().Token);
+		}
+
+		async Task SendNodeList (HttpContext context)
+		{
+			Console.WriteLine ("hello chrome! json/list");
+			try {
+				var response = new JArray (JObject.FromObject (new {
+					description = "node.js instance",
+					devtoolsFrontendUrl = "chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=localhost:9300/91d87807-8a81-4f49-878c-a5604103b0a4",
+					faviconUrl = "https://nodejs.org/static/favicon.ico",
```

Comment:
Can we just omit this? It probably produces an external request for the favicon on every run, if not multiple